### DISCUSSION TITLE
Welcome to Greyhole Discussions!

### DISCUSSION THREAD

**Discussion by gboudreau** [gboudreau/Greyhole#249] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by uSlackr** [comment]
So glad to see this.  I never know where to ask questions.  I hope we can ask things that aren't "issues".  But oif you prefer a different place - just say so.
And thank you for an awesome tool

**Reply by gboudreau** [reply]
Totally the place to ask questions, discuss, suggest.
It also easy to convert a discussion into a issue, and vice-versa, so if you think you found a bug, starting as a discussion to confirm first works too.

**Reply by uSlackr** [reply]
Is there documentation showing how to move a set of drives w/Greyhole to a new system?  I just did this and was somewhat successful, but couldn't find a good guide

**Reply by gboudreau** [reply]
Yes, there is this : https://github.com/gboudreau/Greyhole/wiki/ReuseStoragePool
Improvements suggestions are welcome; this has been written ages ago.

**Comment by Rastalovich** [comment]
Just wanted to say many thanks for maintaining Gh. I've used this over the years and it's been brilliant - does exactly what I want a pool handler of samba shares to do. I'm in the middle of a new project to migrate my existing setup to a vm and since Gh has been so stable on the old server, it's not often I'd have cause to interact ... as it just works! issues I have had in the past have ultimately not been down to Gh but an unnoticed hard disk fault or memory or something. (or that auto-security update of samba that I'll not forget in a hurry cos it blocked my whole sharing platform - but fair enough, you already knew about samba 4 issue and by the time I got onto your site you'd already worked out a solution :))

**Comment by uSlackr** [comment]
Let me chime in to add my thanks.  Been using this since an early Amahi version.  Now running it stand-alone on Debian.  It's been a rock-solid and a huge benefit to my home network config.  Thanks for all the time spent.
